k8s/trillian/manifest/logserver.yaml

```diff
@@ -0,0 +1,71 @@
+apiVersion: v1
+kind: List
+items:
+- apiVersion: v1
+  kind: Service
+  metadata:
+    name: $NAME-logserver-service
+    labels: &LogServerLabels
+      app.kubernetes.io/name: "$NAME"
+      app.kubernetes.io/component: logserver
+    annotations:
+        cloud.google.com/load-balancer-type: "Internal"
+  spec:
+    ports:
+    - name: grpclb
+      port: 8090
+    - name: metrics
+      port: 8091
+    selector: *LogServerLabels
+    type: LoadBalancer
+- apiVersion: apps/v1
+  kind: Deployment
+  metadata:
+    name: $NAME-logserver-deployment
+    labels: *LogServerLabels
+  spec:
+    replicas: $TRILLIAN_LOGSERVER_REPLICAS
+    selector:
+      matchLabels: *LogServerLabels
+    template:
+      metadata:
+        labels: *LogServerLabels
+      spec:
+        restartPolicy: Always
+        containers:
+        - name: trillian-logserver
+          args: [
+          "$(STORAGE_FLAG)",
+          "--storage_system=$(STORAGE_SYSTEM)",
+          "--quota_system=etcd",
+          "--etcd_servers=$NAME-etcd-cluster-client:2379",
```

Comment:
Is the name here correct? This name is not defined in scope of this PR.

Reply:
This service is created by etcd-operator, see https://github.com/coreos/etcd-operator/blob/master/doc/user/client_service.md for more information.